schemas/hub/definitions.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$id": "https://eq.ons.gov.uk/hub/definitions.json",
+  "hub": {
+    "type": "object",
+    "properties": {
+      "enabled": {
+        "description": "Whether or not the hub and spoke pattern is enabled for this questionnaire.",
+        "type": "boolean"
+      },
+      "required_completed_sections": {
+        "description": "The sections that need to be complete prior to the hub displaying",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "complete": {
+        "type": "object",
+        "description": "The title/guidance that will appear on the hub, when the questionnaire is complete",
+        "anyOf": [
+          {
+            "required": ["title"]
+          },
+          {
+            "required": ["guidance"]
+          }
+        ]
+      },
+      "incomplete": {
+        "type": "object",
+        "description": "The guidance that will appear on the hub, when the questionnaire is incomplete.",
+        "properties": {
+          "guidance": {
+            "type": "string"
+          }
+        },
```

Comment:
The `title` and `guidance` keys should be defined as properties outside these `complete/incomplete` keys then referenced by them. It should also be mandatory to provide one of those keys.